src/videojs-contrib-hls.js

```diff
@@ -106,11 +86,17 @@ const getHlsPlaylistIndex = function(media, playlists) {
  */
 const handleHlsMediaChange = function(qualityLevels, playlistLoader) {
   let newPlaylist = playlistLoader.media();
-  let selectedIndex = getHlsPlaylistIndex(newPlaylist, playlistLoader.master.playlists);
+  let i;
```

Comment:
Not a big deal, but I generally like to keep the index vars within the scope of the loop when possible, as outside of it the context is often lost, even when the code is close together. Maybe we can use selectedIndex as the var?